websites/T/Tiermaker/tsconfig.json

```
@@ -0,0 +1,6 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+  }
```

How did this happen?
```suggestion
}
```

---

no idea lol, fixed it